src/studio-header/HeaderBody.jsx

```diff
@@ -110,6 +111,7 @@ const HeaderBody = ({
               iconAs={Icon}
               onClick={searchButtonAction}
               aria-label={intl.formatMessage(messages['header.label.search.nav'])}
+              alt={intl.formatMessage(messages['header.label.search.nav'])}
```

Comment:
Not related to this feature. This is a fly-by fix of a console warning. Let me know if I should create a different PR for this.

Reply:
Thanks for fixing these! Fewer warnings is always good.